assets/training/finetune_acft_image/environments/acpt_image_mmdetection/context/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#PTCA image
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu117-py38-torch1131:{{latest-image-tag:vision\.biweekly\.\d{6}\.\d+}}
+
+USER root
+RUN apt-get -y update
+
+RUN pip install azureml-metrics==0.0.24
```

Comment:
I would recommend to have all dependencies in a single requiremetns txt

Reply:
Done